workshop/static/assets/bootstrap.sh

```diff
@@ -60,13 +64,18 @@ function install_utility_tools() {
 function install_linuxbrew() {
     _logger "[+] Creating touch symlink"
     sudo ln -sf /bin/touch /usr/bin/touch
-    _logger "[+] Installing homebrew..."
-    echo | sh -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
-    _logger "[+] Adding homebrew in PATH"
-    test -d ~/.linuxbrew && eval $(~/.linuxbrew/bin/brew shellenv)
-    test -d /home/linuxbrew/.linuxbrew && eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)
-    test -r ~/.bash_profile && echo "eval \$($(brew --prefix)/bin/brew shellenv)" >>~/.bash_profile
-    echo "eval \$($(brew --prefix)/bin/brew shellenv)" >>~/.profile
+
+    if [[ $(command -v brew) == "" ]]; then
+        _logger "[+] Installing homebrew..."
+        echo | sh -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+        _logger "[+] Adding homebrew in PATH"
+        test -d ~/.linuxbrew && eval $(~/.linuxbrew/bin/brew shellenv)
```

Comment:
Are the installation paths still `/home/linuxbrew` and `~/.linuxbrew` even though the download URL changed from Linuxbrew to Homebrew?

Reply:
Yes, the homebrew installer checks if the OS is Linux or MacOS. If the OS is Linux and the user has sudo access it installs to `/home/linuxbrew` or `~/.linuxbrew` if the user does not have sudo access. 

These lines will check for either install type so environment variables can be set 
``` 
  test -d ~/.linuxbrew && eval $(~/.linuxbrew/bin/brew shellenv)
  test -d /home/linuxbrew/.linuxbrew && eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)
```

Reply:
Ok, thanks. I will merge now.